app/models/featured_collection_list.rb

```diff
@@ -22,6 +22,8 @@ def featured_collections
       collection.destroy if collection.presenter.blank?
       collection.presenter.blank?
     end
+    sort_by_title! unless manually_ordered?
```

Comment:
If there are specs for #featured_collection this may be a good time to add a context to it.

Reply:
added specs!

Reply:
thanks it looks great!